src/faldbt/project.py

```diff
@@ -286,15 +297,22 @@ def _find_features(self) -> List[Feature]:
             for column_name in model.columns.keys():
                 if column_name == model.meta[keyword]["feature_store"]["entity_column"]:
                     continue
-                if column_name == model.meta[keyword]["feature_store"]["timestamp_column"]:
+                if (
+                    column_name
+                    == model.meta[keyword]["feature_store"]["timestamp_column"]
+                ):
```

Comment:
Maybe we can put `model.meta[keyword]["feature_store"]` in a variable to avoid multiline statements like this